customer-segment-template-extension/locales/en.default.json

```diff
@@ -1,4 +1,4 @@
 {
-    "name": "First-time customers who bought specific products",
+    "title": "First-time customers who bought specific products",
```

Comment:
About this rename https://github.com/Shopify/extensions-templates/blob/main/customer-segment-template-extension/shopify.extension.toml.liquid#L5 is still `t:name`. I'm wondering if it makes sense to use the translation there. I know @vividviolet talked about something there and the `description` support being missing. I'm wondering if something evolved there.

I'm thinking it looks weird if to use this as an app title if we encourage multiple templates per `ts` file. 🤔

Reply:
I agree, the t:name comes from the CLI command when naming the extension. I changed it up so now the extension name is independent of the template title. I thought this was a better solution, at least until there is proper support for the description field.